.readme-partials.yml

```diff
@@ -25,3 +25,29 @@ body: |-
 
   The `@google-cloud/logging` library will handle batching and dispatching
   these log lines to the API.
+
+  ## Writing to Stdout
+
+  Serverless applications like Cloud Functions, Cloud Run, and App Engine
+  should use the `LogSync` class to synchronously log to `stdout` or another
+  custom transport. Logs written to `stdout` are picked up by a [Logging agent](https://cloud.google.com/logging/docs/agent/logging)
```

Comment:
The logging agent you refer to here is not the logging agent that is picking up the logs in serverless scenarios. Can you rephrase this to mention that it's picked out-of-process by a logging agent in respective serverless environment?

Reply:
yeah good pt, lemme say 'or'